tests/unit/util/test_helper_add_field.py

```diff
@@ -163,8 +120,12 @@ def test_add_list_with_nested_keys(self):
                 }
             }
         }
-        add_was_successful = add_field_to(
-            testdict, "key1.key2.key3.key4.key5.list", ["content"], extends_lists=True
-        )
-        assert add_was_successful
+        add_field_to(testdict, "key1.key2.key3.key4.key5.list", ["content"], extends_lists=True)
         assert testdict == expected
+
+    def test_add_value_not_as_list_if_it_is_a_new_value_even_though_extends_lists_is_true(self):
+        document = {
+            "some": "field",
+        }
+        add_field_to(document, "new", "list", extends_lists=True)
+        assert document.get("new") == "list"
```

Comment:
```suggestion
    def test_add_value_not_as_list_if_it_is_a_new_value_even_though_extends_lists_is_true(self):
        document = {
            "some": "field",
        }
        add_field_to(document, "new", "list", extends_lists=True)
        assert document.get("new") == "list"
```

consider refactoring the test name and add a comment on for the purpose of this test